modules/monitoring-assigning-tolerations-to-monitoring-components.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc
+
+[id="assigning-tolerations-to-monitoring-components_{context}"]
+= Assigning tolerations to monitoring components
+
+You can assign tolerations to any of the monitoring stack components to enable moving them to tainted nodes.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Start editing the `cluster-monitoring-config` ConfigMap:
++
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Specify `tolerations` for the component:
++
+[source,yaml,subs=quotes]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    *_component_*:
+      tolerations:
+        *_tolerations specification_*
+----
++
+Substitute `<component>` and `<toleration specification>` accordingly.
++
+For example, a `kubectl taint nodes node1 key1=value1:NoSchedule` taint prevents the scheduler from placing pods in `foo: bar`. To make the `alertmanagerMain` component ignore that taint and to place `alertmanagerMain` in `foo: bar` normally, use this toleration:
```

Comment:
`kubectl taint nodes node1 key1=value1:NoSchedule`
=>
`oc adm taint nodes node1 key1=value1:NoSchedule`

prevents the scheduler from placing pods in `foo: bar`
=>
prevents the scheduler from placing pods in `foo: bar` node

Reply:
Implemented in #16978.